tests/OmniSharp.DotNet.Tests/project.json

```diff
@@ -0,0 +1,24 @@
+{
+    "version": "1.0.0-*",
+    "compilationOptions": {
+        "warningsAsErrors": true
+    },
+    "dependencies": {
+        "OmniSharp.DotNet": "1.0.0-*",
+        "TestCommon": "1.0.0-*",
+        "xunit": "2.1.0"
+    },
+    "frameworks": {
+        "netstandardapp1.5": {
```

Comment:
Is it best to use netstandardapp or netcoreapp here? I think it would be best to use the same everywhere to avoid confusing people.